### PR TITLE
Docs/776 Documentation update for LBS support

### PR DESCRIPTION
**Pull request by danigian**
# PR for issue #776 
## What is being addressed

This PR is addressing some legacy documentation on Packet Forwarder and including a table for additional custom configuration of the Basics Station IoT Edge Module.

### CODE REVIEW

**Comment by danigian** [comment]
For the reviewer: this PR is not taking into account any future change to the "Deploy to Azure" functionality via the templates in [this folder](https://github.com/Azure/iotedge-lorawan-starterkit/tree/dev/Template)

File: docs/quickstart.md [hunk 1]
```diff
@@ -57,9 +57,9 @@ By using the `docker ps` command, you should see the Edge containers being deplo
...
+The template provision an IoT Hub with a [LoRa Basics™ Station](https://github.com/lorabasics/basicstation) and a network server module already pre-configured to work out of the box. As soon as you connect your IoT Edge device in point 4 above, those will be pushed on your device. You can find template definition and Edge deployment specification [here](https://github.com/Azure/iotedge-lorawan-starterkit/tree/dev/Template).
 
-If you are using the the RAK833-USB, you'll need to adjust the template to use the right LoRaWan Packet Forwarder. You will find a full documentation in the LoRaWanPktFwdRAK833USB [submodule](https://github.com/Azure/iotedge-lorawan-starterkit/tree/dev/Docs/).
+If you are using the the RAK833-USB, you'll need to build a different LoRa Basics™ Station image. You can find a fork of the official Basic Station repository with support for RAK833-USB [here](https://github.com/danigian/basicstation).
```
Comment (Mandur) [inline docs/quickstart.md:62]: could we remove the submodule then please :p

File: docs/user-guide/station-module-configuration.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Basics Station IoT Edge Module configuration parameters
+
+The following table is providing a list of configuration options, to be provided as environment variables for manual configuration of the 'LoRaBasicsStationModule':
+
+| Environment variable name | Description                                                  | Mandatory                                                    |
+| ------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| TC_URI                    | The URI to the LNS Server implementation                     | **Yes** (i.e.: 'ws://IPorDNS:PORT')                          |
```
Comment (Mandur) [inline docs/user-guide/station-module-configuration.md:7]: what is meant here by IPorDNS?

Comment (Mandur) [reply docs/user-guide/station-module-configuration.md:7]: Oh... I understand now... maybe IP_or_DNS?

**Review by Mandur — APPROVED** [review]

File: docs/quickstart.md (hunk 1 above)
Comment (danigian) [reply docs/quickstart.md:62]: Will do in a separate PR

**Pull request merged**